proposal/indented-syntax-improvements.md

```diff
@@ -0,0 +1,215 @@
+    url("leatherman-outline.woff") format("woff")
+  )
+```
+
+Alternatively, authors can use Scss block syntax with braces and semicolons
+within the indented document.
+
+```sass
+a
+  color: blue
+
+.grid {
+  display: grid;
+  grid-template:
+    'logo title copy' auto
+    'alert alert alert' minmax(0, auto) / 1fr 1fr 1fr;
+}
+```
+
+### Ending statements
+
+Semicolons will always cause a statement break. If one occurs in a context where
+a statement can not end, an error will be thrown.
+
+### Places where a line break must create a statement break
+
+By design, line breaks are ignored as meaningless white space, except in
+contexts where the semantics define that a statement may end. When an author
+inserts a line break they intend to be meaningless in a position where a
+statement may end, the statement will end, and parsing will most likely fail on
+the subsequent text. The following places are where line breaks will cause
+statements to end.
+
+#### After a SassScript value
+
+In a simple declaration, `$foo: bar \n`, the line break must cause the statement
+to end.
+
+This may be surprising in more complex situations, for instance, with binary
+operators. `$foo: 3\n+ 4\n` ends the statement after `3`, but `$foo: 3 +\n4\n`
+ends the statement after `4`. Wrapping with the order of operations operator
+`()` allows authors more flexibility with `$foo: (3\n+ 4)`.
+
+This also applies to flow control at-rules. `@if $a \n and $b` would end the
+statement after `$a`, but `@if ($a \n and $b)` can be parsed.
+
+#### After a non-enclosed list begins
+
+A line break in a list that is not in a `BracketedListExpression` or enclosed in
+`()` must cause a statement break.
+
+`$var: 1 2\n3` and `$var: 1, 2\n, 3` can not be parsed to determine when the
+statement has ended. Alternates `$var: (1 2\n3)`, `$var: [1 2\n3]`, and `$var:
+(1, 2,\n 3)` can be parsed.
+
+Comma separated lists can not use a trailing comma to signify that a list will
+continue after the line break, as this would break existing stylesheets with
+trailing commas.
+
+Because arguments to functions and mixins are already wrapped in `()`, line
+breaks in arguments do not need to cause a statement break. Interpolations are
+wrapped in `#{}` so line breaks do not need to end statements.
+
+#### At-rules
+
+For any at-rule that is supported by native CSS, line breaks after the `@` and
+before a block or statement are not supported. This includes `@include`,
+(which overlaps with Sass), `@supports`, `@media`, `@keyframes` and any unknown
+at-rule.
+
+These rules should be emitted as is, with no special handling from Sass.
+
+### Design Decisions
+
+While some CSS at-rules may have contexts where a line break would not be
+meaningful, custom handling of line breaks is outside of the scope of this
+proposal. For instance, `@media (hover: hover) and \n (color)` is not supported,
+even though line breaks do not end statements after boolean operators in general
+SassScript.
```

Comment:
I think it's worth explaining that this is because we don't want Sass's behavior for at-rules to have to depend on us adding support for newer CSS syntax.